examples/nemo/eg3/README

```diff
@@ -51,6 +54,9 @@ PSyclone's Schedule view of the code after adding OpenACC Kernels and
 Data directives, and 5) the transformed Fortran code with the OpenACC
 directives added.
 
+Doing the same thing with kernels_managed_mem_trans will produce similar
```

Comment:
Running PSyclone with the `kernels_managed_mem_trans.py` script will ...

Reply:
Done.

Reply:
I've updated slightly and am now happy :-)